voice/src/main/kotlin/gateway/DefaultVoiceGateway.kt

```diff
@@ -196,7 +197,7 @@ public class DefaultVoiceGateway(
                     val copy = command.copy(data = command.data.copy(address = "ip"))
                     "Voice Gateway >>> ${Json.encodeToString(Command.SerializationStrategy, copy)}"
                 }
-                is Heartbeat, is Resume, is SendSpeaking -> "Voice Gateway >>> $json"
+                else -> "Voice Gateway >>> $json"
```

Comment:
Please don't use `else` here to prevent a future command with sensitive data being logged.